src/file.ts

```diff
@@ -2043,14 +2043,27 @@ class File extends ServiceObject<File> {
     delete options.destination;
 
     const fileStream = this.createReadStream(options);
-
+    let receivedData = false;
     if (destination) {
-      fileStream.on('error', callback).once('data', data => {
-        // We know that the file exists the server - now we can truncate/write to a file
-        const writable = fs.createWriteStream(destination);
-        writable.write(data);
-        fileStream.pipe(writable).on('error', callback).on('finish', callback);
-      });
+      fileStream
```

Comment:
[Optional]: Now we're on Node 12+ we can use `for await (const chunk of readable)` to make this tinier: https://nodejs.org/api/stream.html#readablesymbolasynciterator

Reply:
Since this isn't within an async function currently I'm going to leave it alone for the time being until we modernize our function signatures.